.github/workflows/test.yml

```diff
@@ -0,0 +1,28 @@
+name: Unit Tests
+
+on: push
+
+jobs:
+  test:
+    name: Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 # step 1. checkout repository
+      - uses: actions/setup-node@v1 # step 2. install node
+        with:
+          node-version: "14.x"
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+      - uses: actions/cache@v2 # step 3. install cache
+        id: yarn-cache # check for yarn cache
+        with:
+          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner  .os }}-yarn-
+      - run: yarn # step 5. install packages
+        working-directory: server
+      - run: yarn test:unit # step 6. run unit tests
```

Comment:
Right now our `yarn test:unit` script is testing Typescript with `ts-jest` instead of raw,  compiled JS. This isn't the best solution because on production our server will be running vanilla JS. We can start working on this with another PR, but I suggest we get this merged in so we at least have some sort of protection over the quality of the `main` branch.